piksi_tools/console/settings.yaml

```diff
@@ -290,9 +290,8 @@
   enumerated possible values: True,False
   Description: Enable Skylark client.
   Notes: |
-      If True, Skylark client will be used. Klobuchar correction must be disabled
-      if order for Skylark to function properly, set solution.disable_klobuchar_correction
-      to True before enabling Skylark.
+      If True, Skylark client will be used. Klobuchar corrections are disabled in order for Skylark to function properly.
```

Comment:
If we've removed klobuchar corrections as a setting, can we just remove the references to it? @anth-cole

Reply:
I'd agree - let's just make this setting say `If True, Skylark client will be used for reference observations`